sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -235,12 +235,12 @@ System.out.printf("Detected language name: %s, ISO 6391 name: %s, score: %.2f.%n
     detectedLanguage.getName(), detectedLanguage.getIso6391Name(), detectedLanguage.getScore());
 ```
 
-### Recognize categorized entity
+### Recognize entity
 <!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L123-L126 -->
 ```java
 String text = "Satya Nadella is the CEO of Microsoft";
 textAnalyticsClient.recognizeEntities(text).forEach(entity ->
-    System.out.printf("Recognized categorized entity: %s, category: %s, subCategory: %s, score: %.2f.%n",
+    System.out.printf("Recognized entity: %s, category: %s, subCategory: %s, score: %.2f.%n",
```

Comment:
During the arch board, Assaf mentioned that we should remove the fixed points for score as the service will return them.